### ISSUE TITLE
Support for more payment methods

### ISSUE THREAD

**Issue by gtandersen** [runbox/runbox7#1332]
**Is your feature request related to a problem? Please describe.**
Customers increasingly make use of new payment methods.

**Describe the solution you'd like**
Support for more payment methods such as SEPA and Boleto via our payment processor Stripe.

According to https://dashboard.stripe.com/settings/payment_methods:
`
We've detected that you are using payment_method_types to configure payment methods. To manage payment methods from this page, [replace payment_method_types with automatic_payment_methods](https://stripe.com/docs/payments/payment-element/migration#one-time-payment-intent). `

In other words, supporting additional payment methods via the Stripe dashboard requires a small change to our payment code.

**Comment by castaway** [comment]
This has been implemented by upgrading our Stripe implementation to use the Payments element (instead of individual Card fields), now Stripe will show all payment methods available for the chosen currency/country)
